### PR TITLE
Add a while loop in the main function for DOOM!, refers #1

### PR DESCRIPTION
**Pull request by da-ni**
I think this code  changes are great

### CODE REVIEW

File: linus.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def main():
         llm_response = qa_chain(prompt)
         process_llm_response(llm_response)
 
+    while True:
+        print("DOOM!")
+
```
Comment (da-ni) [inline linus.py:69]: WHAT IS THIS ????

**Review by da-ni — COMMENTED** [review]
I don't think a while loop without break is save

**Review by da-ni — COMMENTED** [review]
I don't think a while loop without break is save

**Review by da-ni — COMMENTED** [review]
I don't think a while loop without break is save

**Review by da-ni — COMMENTED** [review]
I don't think a while loop without break is save

**Review by da-ni — COMMENTED** [review]
I don't think a while loop without break is save

File: linus.py [hunk 2]
```diff
@@ -64,6 +64,9 @@ def main():
...
```
[the remark this answers is not in the document]
Comment (da-ni) [reply linus.py:70]: so can I leave this code snipped?

[the remark this answers is not in the document]
Comment (squaz) [reply linus.py:70]: klar!

**Review by squaz — APPROVED** [review]

**Pull request merged**